Makefile

```diff
@@ -13,7 +13,7 @@ IMAGE_TAG_PREFIX ?=
 
 ## dockerfile for each container default to alpine base image
 DOCKERFILE_BACKEND ?= Dockerfile.backend-container
-DOCKERFILE_FRONTEND ?= Dockerfile.frontend-container
+DOCKERFILE_FRONTEND ?= frontend/Dockerfile.frontend-container
 
 BINARIES=tornjak-backend tornjak-manager
```

Comment:
What about the binaries?

Reply:
the go binaries are built in a directory `bin/` (which is gitignored). 
i would personally keep it as `tornjak-xxx` but could go either way